subsys/net/ip/Kconfig

```diff
@@ -10,11 +10,16 @@ menu "IP stack"
 	depends on NETWORKING
 
 # Hidden option
-config NET_L2_RAW_CHANNEL
+config NET_RAW_MODE
 	bool
 	default n
+	help
+	This is a very specific option used to built only the very minimal
```

Comment:
Should we describe here that
* user should not set this normally
* what applications (wpan_serial etc) use this and what purpose these applications set this value

Reply:
I documented it a bit better, but the more important one was in ieee802154 where I reference wpan* stuff.